docs/diagnostics/PX1035.md

```diff
@@ -8,12 +8,23 @@ This document describes the PX1035 diagnostic.
 | PX1035 | The DAC has multiple key declarations which use the same set of fields | Warning (ISV Level 3: Informational) | Available | 
 
 ## Diagnostic Description
-A DAC cannot have multiple primary, foreign, or unique key declarations which include the same fields. The warning is shown only for DACs that have the `PXCacheName` or `PXPrimaryGraph` attribute.
+A DAC should not have multiple primary, foreign, or unique key declarations which include the same fields. 
+The warning is shown only for DACs that have the `PXCacheName` or `PXPrimaryGraph` attribute.
 The diagnostic does not check abstract DACs and DACs which have only unbound DAC properties (fully-unbound DACs used for custom popups and filters for inquiry forms).
 
-To fix the issue, you should remove all primary, foriegn, or unique key declarations except one.
+The diagnostic also checks the declaration and implementation of a unique key by the following rules:
```

Comment:
PX1035 diagnostic checks for duplicate keys only, all these rules must be moved to PX1036